model/pyproject.toml

```diff
@@ -16,6 +16,7 @@ jax = "^0.4.24"
 numpy = "^1.26.4"
 polars = "^0.20.13"
 pillow = "^10.3.0" # See #56 on CDCgov/multisignal-epi-inference
+numpydoc = "^1.7.0"
```

Comment:
Move numpydoc to the `tool.poetry.group.dev`.

Reply:
Numpydoc is only needed for checks, so it should be moved out of this group to dev.

Reply:
Addressed in f71bcda